Google.PowerShell/Common/ActiveUserToken.cs

```diff
@@ -0,0 +1,118 @@
+﻿using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using System;
+using System.IO;
+
+namespace Google.PowerShell.Common
+{
+    /// <summary>
+    /// OAuth 2.0 model for a successful access token response as specified in 
+    /// http://tools.ietf.org/html/rfc6749#section-5.1.
+    /// </summary>
+    public class ActiveUserToken
+    {
+        /// <summary>
+        /// The user that this token corresponds to.
+        /// </summary>
+        public string User;
+
+        /// <summary>The access token issued by the authorization server.</summary>
+        public string AccessToken { get; internal set; }
```

Comment:
Make this setter `private` as well as the one on line 27. (Unless there is a reason for other types to modify the field within the same assembly?)